src/nodes/act_external_canister/act_external_canister_method.rs

```diff
@@ -11,14 +11,19 @@ pub struct ActExternalCanisterMethod {
 }
 
 impl ActExternalCanisterMethod {
-    pub fn to_token_stream(&self, canister_name: &String) -> TokenStream {
-        let call_function = self.generate_call_function(canister_name);
-        let call_with_payment_function = self.generate_call_with_payment_function(canister_name);
+    pub fn to_token_stream(
+        &self,
+        canister_name: &String,
```

Comment:
Implement the ToTokenStream trait and combine these params into one context struct